training/src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -51,16 +41,25 @@ public Book findOne(@PathVariable Long id) {
             .orElseThrow(BookNotFoundException::new);
     }
 
-    @PostMapping
+    @PostMapping("/createBook")
```

Comment:
Usually, I think that by convention, when we are talking about a creation endpoint, it is left just with `/` and you know it's a creation endpoint because of the `post mapping.`